spec/index.bs

```diff
@@ -5885,9 +5885,17 @@ details.
                 [=interpolation=]
                 of the input.
             - There is less than |device|.limits.{{supported limits/maxInterStageShaderComponents}}
-                components of user-defined outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
+                components of user-defined and built-in outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
```

Comment:
nit: instead of duplicating the validation, I think it would be simpler in the spec to have a counter that we add 1 to if the primitive topology is point-list. But we can do that later (and imho we could separate all the validation rules to a "validating inter-stage interfaces" step)

Reply:
Done. PTAL, thanks!